tajo-catalog/tajo-catalog-server/src/main/java/org/apache/tajo/catalog/dictionary/InfoSchemaMetadataDictionary.java

```diff
@@ -110,7 +110,7 @@ private TableDescriptor getTableDescriptor(String tableName) {
     return tableDescriptor;
   }
   
-  public CatalogProtos.TableDescProto getTableDesc(String tableName) {
+  public CatalogProtos.TableDescProto getTableDesc(String tableName) throws UndefinedTableException {
```

Comment:
Both ```getTableDesc()``` and ```existTable()``` internally use ```getTableDescriptor()```, so seem to be changed according to the implementation of ```getTableDescriptor()```. Please consider the above comment.